src/main/resources/beam-template.conf

```diff
@@ -280,6 +272,21 @@ beam.physsim.inputNetworkFilePath = ${beam.inputDirectory}"/r5/physsim-network.x
 beam.physsim.events.fileOutputFormats = "csv" # valid options: xml(.gz) , csv(.gz), none - DEFAULT: csv.gz
 beam.physsim.events.eventsToWrite = "ActivityEndEvent,ActivityStartEvent,LinkEnterEvent,LinkLeaveEvent,PersonArrivalEvent,PersonDepartureEvent,VehicleEntersTrafficEvent,VehicleLeavesTrafficEvent"
 
+beam.physsim.skipPhysSim = false
+beam.physsim.physSimType = "string | JDEQSim"
+
+# JDEQ
+beam.physsim.jdeqsim.agentSimPhysSimInterfaceDebugger.enabled = false
+beam.physsim.jdeqsim.cacc.enabled = false
+beam.physsim.jdeqsim.cacc.minRoadCapacity = "int | 2000"
+beam.physsim.jdeqsim.cacc.minSpeedMetersPerSec = "int | 20"
+beam.physsim.jdeqsim.cacc.speedAdjustmentFactor = "double | 1.0"
+beam.physsim.jdeqsim.cacc.capacityPlansWriteInterval = 0
+beam.physsim.jdeqsim.cacc.adjustedMinimumRoadSpeedInMetersPerSecond = "double | 1.3"
+
+# CCH
+beam.physsim.cch.congestionFactor = "double | 1.0"
```

Comment:
Can be left as 1 (automatically correct up scaling to 100% done internally based on agentSampleSizeAsFractionOfPopulation). However the simulated population can be further tuned using this parameter.